lib/options.js

```diff
@@ -49,7 +49,12 @@ var defaults = {
 
     // If you want esprima not to throw exceptions when it encounters
     // non-fatal errors, keep this option true.
-    tolerant: true
+    tolerant: true,
+    
+    // If you want to override the quotemarks used in string literals, specify
+    // either "single" or "double" here
+    // Otherwise, the input marks will be preserved
+    quotemark: null,
```

Comment:
Maybe we could aim to some parity with escodegen? they use the key "quotes" instead of "quotemark" and "single",  "double" or "auto" as options. see https://github.com/estools/escodegen/wiki/API#optionformatquotes